assets/styles/04-components/tooltip.scss

```
@@ -2,6 +2,7 @@ tool-tip-container {
   position: relative;
   display: inline-block;
   cursor: help;
+  text-decoration: dashed underline;
```

```suggestion
  text-decoration: 1px var(--theme-color--brand) wavy underline;
```

A bit tight and heavy as it was. How about this?

---

Love it! Thanks!